go.mod

```diff
@@ -21,7 +25,10 @@ require (
 	github.com/xeipuuv/gojsonschema v0.0.0-20170210233622-6b67b3fab74d
 	golang.org/x/net v0.0.0-20191027093000-83d349e8ac1a
 	golang.org/x/sync v0.0.0-20181221193216-37e7f081c4d4
-	golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a
+	golang.org/x/sys v0.0.0-20190412213103-97732733099d
+	gopkg.in/jcmturner/goidentity.v3 v3.0.0 // indirect
+	gopkg.in/jcmturner/gokrb5.v7 v7.3.0 // indirect
```

Comment:
Is it right that you have v7 when [go-spnego is using v5.3.0](https://github.com/dpotapov/go-spnego/blob/master/go.mod#L11)?

Reply:
It appears that's a mistake.  I think #4035 should fix that.

It perhaps snuck in before I used go-spnego and was doing things by hand.  Thanks for pointing it out.